python-package/setup.py

```diff
@@ -181,9 +225,11 @@ def run(self):
 
 if __name__ == "__main__":
     if (8 * struct.calcsize("P")) != 64:
-        raise Exception('Cannot install LightGBM in 32-bit python, please use 64-bit python instead.')
+        raise Exception('Cannot install LightGBM in 32-bit Python, please use 64-bit python instead.')
 
     dir_path = os.path.dirname(os.path.realpath(__file__))
+    path_log = os.path.join(os.path.expanduser('~'), 'LightGBM compilation.log')
```

Comment:
I think it is better to use `LightGBM_compilation.log` instead of `LightGBM compilation.log` because filenames with spaces are not very friendly for command line users